src/main/java/core/basesyntax/models/animals/Penguin.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.models.animals;
+
+import core.basesyntax.models.interfaces.Swimable;
+
+public class Penguin extends Animal implements Swimable {
+    private String name;
+    private int age;
```

Comment:
these fields are duplicated in several methods, let's move them to our abstract class, to avoid code duplication

Reply:
moved